more/free-programming-cheatsheets.md

```diff
@@ -181,3 +192,10 @@
 * [MySQL Cheatsheet](https://s3-us-west-2.amazonaws.com/dbshostedfiles/dbs/sql_cheat_sheet_mysql.pdf) - Database Star (PDF)
 * [PostgreSQL Cheatsheet](https://s3-us-west-2.amazonaws.com/dbshostedfiles/dbs/sql_cheat_sheet_pgsql.pdf) - Database Star (PDF)
 * [SQL Cheatsheet](https://www.codecademy.com/learn/learn-sql/modules/learn-sql-manipulation/cheatsheet) - Codecademy (SQL)
+
+
+### TensorFlow
+
+* [Collection of 10 Tensorflow Cheat Sheets for Machine Learning](https://blog.finxter.com/tensorflow-cheat-sheets/#Cheat_Sheet_2_Altoros) (HTML site with PDF links)
```

Comment:
we don't include meta-lists

Reply:
Maybe better use resources fron altoros website

https://www.altoros.com/visuals/